src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -3097,7 +3188,8 @@ namespace SimAirServingZones {
                           AllowWarmRestartFlag);
 
         // Detect whether the speculative warm restart feature is supported by each controller on this air loop.
-        AirLoopControlInfo(AirLoopNum).AllowWarmRestartFlag = AirLoopControlInfo(AirLoopNum).AllowWarmRestartFlag && AllowWarmRestartFlag;
+        if (AirLoopNum > 0)
+            AirLoopControlInfo(AirLoopNum).AllowWarmRestartFlag = AirLoopControlInfo(AirLoopNum).AllowWarmRestartFlag && AllowWarmRestartFlag;
```

Comment:
This code works fine, but it can cause visual indentation issues.  It looks like you've started a new block since the statement portion of the IF statement is indented on the next line.  But it's actually not.  I would prefer to see brackets used here to avoid confusion.

Reply:
@Myoldmopar Done. Thanks.